docs/changes/v11.0.0/Packaging-Changes.adoc

```diff
@@ -0,0 +1,26 @@
+= Packaging Changes =
+
+== Package Name Changes ==
+
+In PKI 11 the following packages have been renamed:
+
+* `pki-base` -> `dogtag-pki-base`
+* `pki-base-java` -> `dogtag-pki-base-java`
+* `pki-javadoc` -> `dogtag-pki-javadoc`
+* `python3-pki` -> `python3-dogtag-pki`
+* `pki-tools` -> `dogtag-pki-tools`
+* `pki-symkey` -> `dogtag-pki-symkey`
+* `pki-server` -> `dogtag-pki-server`
+* `pki-ca` -> `dogtag-pki-ca`
+* `pki-kra` -> `dogtag-pki-kra`
+* `pki-ocsp` -> `dogtag-pki-ocsp`
+* `pki-tks` -> `dogtag-pki-tks`
+* `pki-tps` -> `dogtag-pki-tps`
+* `pki-acme` -> `dogtag-pki-acme`
+* `pki-server-theme` -> `dogtag-pki-server-theme`
+* `pki-tests` -> `dogtag-pki-tests`
+
+== PKI Console Changes ==
+
+PKI Console is no longer supported in PKI 11, so the `pki-console` and `pki-console-theme` packages have been dropped.
```

Comment:
This seems like it is significant enough to at least be mentioned in the commit message, if not in a separate commit.

Reply:
The console was actually already [dropped](https://github.com/dogtagpki/pki/commit/a6a24bcc0a45b0c18ec983536702f16ecf526941) previously. This PR is just documenting the changes for the end users, in addition to renaming the remaining packages.

We have an [option](https://github.com/dogtagpki/pki/commit/81adacee802bbcf53b7a5381dd5e350ad62f3976) to build console if needed for development, but eventually it will go away too, and from end user's perspective the console is already gone, so I'd rather not mention that option here.

Reply:
This explains why I couldn't find any evidence of it in the diff :thinking: :+1: thanks for the explanation!